libraries/chain/include/eosio/chain/types.hpp

```diff
@@ -100,23 +100,21 @@ namespace eosio::chain {
     */
    class shared_blob : public shared_string {
       public:
-         shared_blob() = delete;
          shared_blob(shared_blob&&) = default;
 
          shared_blob(const shared_blob& s) = default;
 
-
          shared_blob& operator=(const shared_blob& s) = default;
 
          shared_blob& operator=(shared_blob&& ) = default;
 
          template <typename InputIterator>
-         shared_blob(InputIterator f, InputIterator l, const allocator_type& a)
-         :shared_string(f,l,a)
+         shared_blob(InputIterator f, InputIterator l)
+         :shared_string(f,l)
          {}
 
-         shared_blob(const allocator_type& a)
-         :shared_string(a)
+         shared_blob()
+         :shared_string()
          {}
```

Comment:
No, because the `shared_blob` class doesn't have a constructor taking std::string (even though its base class does). I thought of adding one, but I thought this was the least intrusive change.

Reply:
Yes, updated.